### ISSUE TITLE
Arcface/train.parall.py couldn't print acc and lossvalue

### ISSUE THREAD

**Issue by LiaoSS07** [deepinsight/insightface#1540]
I have enabled the eval_metric,  but there is no any acc ad lossvalue during training.

"""
    metric1 = AccMetric()
    eval_metrics = [mx.metric.create(metric1)]
    if config.ce_loss:
        metric2 = LossValueMetric()
        eval_metrics.append(mx.metric.create(metric2))
    model.fit(
        train_dataiter,
        begin_epoch=begin_epoch,
        num_epoch=999999,
        eval_data=val_dataiter,
        eval_metric = eval_metrics,
        kvstore=args.kvstore,
        optimizer=[opt, opt_fc7],
        #optimizer_params = optimizer_params,
        initializer=initializer,
        arg_params=arg_params,
        aux_params=aux_params,
        allow_missing=True,
        batch_end_callback=_batch_callback,
        epoch_end_callback=epoch_cb)
"""

Look forward your help!

**Comment by Insomnia501** [comment]
他的parall_module_local_v1.py里面关于update_metric的东西注释掉了...可能要看着改改？
